tests3/pgtests.py

```diff
@@ -556,7 +556,15 @@ def test_cnxn_set_attr(self):
         SQL_MODE_READ_ONLY   = 1
         self.cnxn.set_attr(SQL_ATTR_ACCESS_MODE, SQL_MODE_READ_ONLY)
 
+
     def test_columns(self):
+        driver_version = tuple(
+            int(x) for x in self.cnxn.getinfo(pyodbc.SQL_DRIVER_VER).split(".")
+        )
+
+        def _get_column_size(row):
+            return row.column_size if driver_version >= (13, 2, 0) else row.precision
```

Comment:
Is it worth putting in a comment here?  Such as:
```python
# the driver changed the name of the returned columns in version 13.02.
# see https://odbc.postgresql.org/docs/release.html, release 13.02.0000, change 6.
```

Reply:
Done.